languages/csharp/reference/exercise-concepts/high-scores.md

```diff
@@ -0,0 +1,33 @@
+# Concepts of high-scores
+
+[Example implementation](https://github.com/exercism/csharp/blob/master/exercises/high-scores/Example.cs)
+
+## General concepts
```

Comment:
Some general concepts I think are missing:

- assignment
- immutability: the readonly modifier
- enumerables: lists can can be iterated over as an enumerable (using Linq)

https://github.com/exercism/v3/blob/master/languages/csharp/reference/exercise-concepts/book-store.md#approach-switch

Reply:
I tried to address your issues with those three categories. Hopefully is this version better.

Reply:
Yep it's nice. Just a FYI: usually it is up to the PR submitter to fix these things, which can help the PR submitted better keep track of what is happening.

Reply:
I'm sorry, I didn't know that. I tried to do it as a pull request, but it gave an error for an unknown URL. I'll keep this in mind for future reference.

Reply:
Sure. No worries.

Reply:
Thanks @ErikSchierboom & @archrisV - added these, and a couple of additional concepts